src/main/java/Poppy.java

```diff
@@ -0,0 +1,155 @@
+import java.util.Scanner;
+
+public class Poppy {
+    public static String input;
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Hello! I'm Poppy");
+        System.out.println("What can I do for you?");
+        Task[] list = new Task[100];
+        input = sc.nextLine();
+        list[0]= new Task(input);
+        int counter = 0;
+        while (!input.equals("Bye")) {
+            String[] str = input.split(" ", 2 );
+            String command = str[0];
+            switch (command) {
```

Comment:
I like the use of switch cases here instead of if cases since there are many different conditions.